app/src/main/kotlin/com/intive/picover/profile/view/ProfileUpdateBottomSheet.kt

```diff
@@ -27,12 +27,11 @@ import com.intive.picover.R
 @OptIn(ExperimentalMaterial3Api::class, ExperimentalComposeUiApi::class)
 @Composable
 fun ProfileUpdateBottomSheet(
-	userName: String,
-	updateUserName: (String) -> Unit,
-	updateName: (String) -> Unit,
+	username: String,
+	saveUsername: () -> Unit,
```

Comment:
In my previous comment I was referring to the view model function so:
- this parameter should be changed on `onSaveClick`
- view model function should be named `saveUserName` instead of `updateName`

Reply:
Done